src/config.ts

```diff
@@ -57,10 +57,10 @@ export const getGitLabEnvVariables = (): Record<string, string> => {
   }
 
   return {
-    mergeRequestBaseSha: process.env.CI_MERGE_REQUEST_DIFF_BASE_SHA as string,
-    gitlabSha: process.env.CI_COMMIT_SHA as string,
-    gitlabToken: process.env.GITLAB_TOKEN as string,
-    projectId: process.env.CI_PROJECT_ID as string,
-    mergeRequestIIdString: process.env.CI_MERGE_REQUEST_IID as string,
+    mergeRequestBaseSha: process.env.CI_MERGE_REQUEST_DIFF_BASE_SHA ?? '',
+    gitlabSha: process.env.CI_COMMIT_SHA ?? '',
+    gitlabToken: process.env.GITLAB_TOKEN ?? '',
+    projectId: process.env.CI_PROJECT_ID ?? '',
+    mergeRequestIIdString: process.env.CI_MERGE_REQUEST_IID ?? '',
```

Comment:
[Note] This is again gross but I want to fix the process.env types properly and in the mean time at least our fix is consistent (this is the fix in another file)